src/javascript/app_2/Services/ws-methods.js

```diff
@@ -42,6 +42,24 @@ const WS = (() => {
     const portfolio = () =>
         BinarySocket.send({ portfolio: 1 });
 
+    const profitTable = (params) => {
```

Comment:
nitpick, `(params)` -> `params`

Reply:
personally, I prefer it the way it is set currently, with `parenthesis`.

Reply:
@ahkum @zakibinary To solve this dilemma we could have a vote and then add it as a rule to eslint (or set up automated code formatting with Prettier 😄 )

Reply:
I mean I'm not totally against the idea, but because there's already a lot of places where we're having `parentheses` around a single argument passed in arrow functions might as well we follow that for now. Plus, I can add enable the `arrow-parens` rule for this in `eslint` for future automation. 😄

Reply:
I am fine with anything, it just bothers me that its not uniformed.

Reply:
It will be uniformed and/or consistent once we enable the `arrow-arens` rule in ESLint. This will enforce consistent use of `parentheses` regardless if the arguments/parameters are single or more. At any rate, @ahkuma , if you're satisfied with our answers or reasoning, kindly mark this issue as resolved.

Reply:
It is now resolved.